src/Catalyst.Core.Lib.Tests/UnitTests/DAO/DaoTests.cs

```diff
@@ -48,45 +50,43 @@ namespace Catalyst.Core.Lib.Tests.UnitTests.DAO
 {
     public class DaoTests
     {
-        private readonly IMapperInitializer[] _mappers;
+        private readonly IMapperInitializer[] _initialisers;
         private readonly HashProvider _hashProvider;
+        private MapperProvider _mapperProvider;
 
         public DaoTests()
         {
             _hashProvider = new HashProvider(HashingAlgorithm.GetAlgorithmMetadata("blake2b-256"));
 
-            _mappers = new IMapperInitializer[]
+            _initialisers = new IMapperInitializer[]
             {
-                new ProtocolMessageDao(),
-                new ConfidentialEntryDao(),
-                new CandidateDeltaBroadcastDao(),
-                new ProtocolErrorMessageSignedDao(),
-                new PeerIdDao(),
-                new SigningContextDao(),
-                new DeltaDao(),
-                new CandidateDeltaBroadcastDao(),
-                new DeltaDfsHashBroadcastDao(),
-                new FavouriteDeltaBroadcastDao(),
-                new CoinbaseEntryDao(),
-                new PublicEntryDao(),
-                new ConfidentialEntryDao(),
-                new TransactionBroadcastDao(),
-                new ContractEntryDao(),
-                new SignatureDao(),
-                new BaseEntryDao(), 
+                new ProtocolMessageMapperInitialiser(),
+                new ConfidentialEntryMapperInitialiser(),
+                new CandidateDeltaBroadcastMapperInitialiser(),
+                new ProtocolErrorMessageMapperInitialiser(),
+                new PeerIdMapperInitialiser(),
+                new SigningContextMapperInitialiser(),
+                new DeltaMapperInitialiser(),
+                new CandidateDeltaBroadcastMapperInitialiser(),
+                new DeltaDfsHashBroadcastMapperInitialiser(),
+                new FavouriteDeltaBroadcastMapperInitialiser(),
+                new CoinbaseEntryMapperInitialiser(),
+                new PublicEntryMapperInitialiser(),
+                new ConfidentialEntryMapperInitialiser(),
+                new TransactionBroadcastMapperInitialiser(),
+                new ContractEntryMapperInitialiser(),
+                new SignatureMapperInitialiser(),
+                new BaseEntryMapperInitialiser(), 
             };
 
-            var map = new MapperProvider(_mappers);
-            map.Start();
+            _mapperProvider = new MapperProvider(_initialisers);
         }
 
-        private TDao GetMapper<TDao>() where TDao : IMapperInitializer => _mappers.OfType<TDao>().First();
+        private TDao GetMapper<TDao>() where TDao : IMapperInitializer => _initialisers.OfType<TDao>().First();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetMapper'.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4404007)